src/main/java/pl/sda/javapoz/model/CountProducts.java

```diff
@@ -1,8 +1,8 @@
 package pl.sda.javapoz.model;
 
-/**
- * Created by RENT on 2017-03-28.
- */
+import javax.persistence.Entity;
```

Comment:
Z tego co pamietam to CountProducts to nie jest encja tylko klasa pomocnicza do wyliczenia produktów, więc chyba tam jednak adnotacji nie powinno być. Może sie trzeba zastanowić nad jakims dziedziczeniem czy interfacem